client/dist/styles.css

```diff
@@ -48,6 +48,109 @@
   z-index: 1;
 }
 
+<<<<<<< HEAD
```

Comment:
This happens to me a lot too with merging! Don't forget to delete '<<<<<<< HEAD'

Reply:
Thats strange, on my vscode I don't see <<<<<HEAD on line 51.  There is a red arrow there though so I don't know if something is going on there.